02Lesson/02Project/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css"/>
+  <title>Intel Project</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/style.css" type="text/css">
 </head>
 <body>
-  <!-- You code start here -->
+  <h1>
+    <em>
+      Daily Planet
+    </em>
+    <nav>
+      <ul>
+        <li>
+          <a href="https://www.foxsports.com/">Sports</a>
+        </li>
+        <li>
+          <a href="https://www.kxan.com/">Local News</a>
+        </li>
+        <li>
+          <a href="https://www.http://www.wsj.com/news/economy/">Economy<a/>
+        </li>
+      </ul>
+    </nav>
+        </h1>

```

Comment:
Element `nav` not allowed as child element `h1`. Indentation of closing `h1` tag is wrong.